nan.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": "nan",
```

Comment:
also add "infinity" process as you defined "is_infinity"?

Reply:
I could think of use cases where you want to set values to NaN as a user, but couldn't find use cases for infinity. I feel like infinity's (there's two of them) are only really the result of computations.

Reply:
well, I even wonder if infinity will happen at all in general openeo usage. Things like division by zero probably cause some kind of ZeroDivision error instead of assigning infinity. Also: most users will probably be fine with receiving NaNs where infinity would theoretically be more correct.

Reply:
Could be. We reference infinity values in divide, ln, log and mod as possible results in supporting environments.